frontend/src/metabase/query_builder/selectors.js

```diff
@@ -314,14 +314,21 @@ export const getLastRunQuestion = createSelector(
     card && metadata && new Question(card, metadata, parameterValues),
 );
 
+export const getOriginalQuestionWithParameters = createSelector(
```

Comment:
Technically this is not `originalQuestion` using the QB terminology. Should it be called something else, e.g. `getQuestionXXX`, i.e. without `original`?

Reply:
Yeah that makes sense, I removed `original` from the selector and any of the references to it

Reply:
let me know if you think `getQuestionWithParameters` is a good name - I keep thinking about it and I have a hunch that it's not the best name, but that might just be me